### DISCUSSION TITLE
Bind Multiple IP's but not All IP's?

### DISCUSSION THREAD

**Discussion by eekay35** [inspircd/inspircd#2018] [Support] [answered]
Hello,

Running InspIRCd v3.15.0

I'm hoping there is a very simple solution/answer to this, but I wasn't able to find any information in the docs or any additional information online.

I have a server with both IPv4 and IPv6 addresses. I'd like to bind the InspIRCd service to a single v4 and single v6 address. I've tried many different variations of this, including:
1) multiple address="" lines within the bind block. One v4 and one v6.
2) multiple IP's in the address="" config separated by whitespace (Ex: address="192.168.1.100 2001:db8::100")
3) multiple IP's in the address="" config separated by comma (Ex: address="192.168.1.100,2001:db8::100")
4) multiple IP's in the address="" config separated by semi-colon (Ex: address="192.168.1.100;2001:db8::100")
etc...

If I specify just one IP (v4 or v6) it works. If I don't specify any IP's (left empty) it binds to all IP's on the host and also works. So, it seems like it is either one IP or all IP's. Is this the expected behavior or am I missing something somewhere? I'd really like to bind to only one v4 and one v6 address if possible, but it appears I may need to bind to all addresses in order for the service to listen on both v4 and v6. Any help would be greatly appreciated!

Regards,
Janketh Jay, III

**Comment by sadiepowell** [comment, marked as answer]
You can specify multiple `<bind>` tags to bind on multiple addresses.

**Reply by eekay35** [reply]
> You can specify multiple `<bind>` tags to bind on multiple addresses.

Thank you, Sadie. I'll give that a try.
